dev-workflow-ui/webContent/view/cleanup.xhtml

```diff
@@ -16,7 +16,7 @@
           <p>#{ivy.cm.co('/cleanup/introduction')}</p>
           <p:panelGrid columns="1" layout="flex">
             <p:selectManyCheckbox id="cleanups" value="#{cleanupBean.cleanupIds}" layout="pageDirection">
-               <f:selectItems value="#{cleanupBean.cleanups}" var="cleanup" itemLabel="#{cleanup.name}" itemValue="#{cleanup.id}"/>
+               <f:selectItems value="#{cleanupBean.cleanups}" var="cleanup" itemLabel="#{ivy.cm.co('/cleanup/'.concat(cleanup.name))}" itemValue="#{cleanup.id}"/>
```

Comment:
May be easier if you do it in the bean? May be move the cleanups one level deeper? E.g. `/cleanup/cleanups/dataCache`?